tests/system/test_client.py

```diff
@@ -722,9 +708,9 @@ def test_numeric_sum(numeric_table):
 
 def test_boolean_casting(alltypes):
     t = alltypes
-    expr = t.groupby(k=t.string_col.nullif('1') == '9').count()
-    result = expr.execute().set_index('k')
-    count = result['count']
+    expr = t.groupby(k=t.string_col.nullif("1") == "9").count()
+    result = expr.execute().set_index("k")
+    count = result["count"]
     assert count.loc[False] == 5840
```

Comment:
I see that this line is failing in the system tests, but must be due to a behavior change in pandas, not this PR.

Reply:
Filed https://github.com/ibis-project/ibis-bigquery/issues/89.

I don't think it should block this PR.